CalculateMethod.java

```diff
@@ -40,6 +40,27 @@ public String calculate(String[] args) {
 			int b = Integer.valueOf(args[1]);
 			return String.valueOf(a % b);
 		}
+	},
+	SIN {
+		@Override
+		public String calculate(String[] args) {
+			Double rad = Double.valueOf(args[0]);
```

Comment:
I think it's much better if you'll use the `double` primitive type rather than the `Double` object class in the `rad` variable